expression/builtin_miscellaneous_test.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
It does be moved from builtin_other.go, but there is no builtin_other_test.go.
We do test these functions in evaluator_test.go. @tiancaiamao